### DISCUSSION TITLE
Kvantum - Fedora 41

### DISCUSSION THREAD

**Discussion by DrCapricorn** [tsujan/Kvantum#1028] [Q&A]
How do I install the most recent version of Kvantum on Fedora 41 - Wayland with all functionality please? I followed install.md and tried what was mentioned here (https://www.reddit.com/r/kde/comments/1bti9o4/how_do_i_apply_kvantum_themes_to_plasma_6/) but I don't know if I can trust the advice of people who seem to make it up like I do. :)

**Comment by tsujan** [comment]
The compilation and installation is clearly explained in `INSTALL.md`.

However, it's highly recommended that you make an installable distro package first (in your case, an rpm package) and then install that package. A direct installation from the source might cause hassles later. If you don't know how to make a correct package, wait for the package maintainers of your distro to make it for you.

As for Wayland, Kvantum doesn't need any adjustment. Make sure you have its [latest version](https://github.com/tsujan/Kvantum/releases).

As to how choose Kvantum for styling Qt apps, it's also explained in `INSTALL.md`. In short, if you use KDE or LXQt, just choose Kvantum from their respective config GUIs; in other DEs, you need a program like `qt6ct`, which should have its own documentation.
